lib/reporters/base.js

```diff
@@ -79,7 +79,7 @@ const BaseReporter = function (formatError, reportSlow, useColors, browserConsol
   }
 
   this.onBrowserError = (browser, error) => {
-    this.writeCommonMsg(util.format(this.ERROR, browser) + formatError(error, '  '))
+    this.writeCommonMsg(util.format(this.ERROR, String(browser)) + formatError(error, '  '))
```

Comment:
This will fix the bug, but ultimately the goal of these calls is to ensure that the browser is named in the message. Wouldn't it be clearer if we just say `browser.name` here?

Reply:
Why assume browser has `name` and that it contains all the relevant information. It probably is so, but since we have a `toString`, I can't think of a valid reason not to use it.